examples/README.md

```diff
@@ -60,3 +67,11 @@ A fast way to get consul is the following:
     2019/07/12 14:41:30 WARN: version 7647 is older or same as the field's Balance
     2019/07/12 14:41:31 INFO: field Balance updated with value 999.99, version: 7649
     2019/07/12 14:41:32 Config: Name: John Doe, Age: 25, City: London, Balance: 999.990000
+
+## 04 Seed values from Vault
+
+    go run examples/04_vault/main.go
+
+    2019/07/18 13:45:45 INFO: field Secret updated with value @pps3cr3t, version: 0
+    2019/07/18 13:45:45 INFO: vault value @pps3cr3t applied on field Secret
```

Comment:
Since this is a secret and we want to hide the secret value from the logs.
To achieve this you can change the value of `Secret` to type `sync.Secret`.